### PR TITLE
[ci skip] Refactor documentation

### PR DESCRIPTION
**Pull request by maxmeyer**
## Summary

This fixes based on [this](https://github.com/cucumber/aruba/issues/453#issuecomment-315591584) comment by @mvz. Additionally it does some minor refactorings to the README

## Details

* Fix example for first-timers (https://github.com/cucumber/aruba/issues/453#issuecomment-315591584)
* Move usage of `aruba` with existing projects to the end to make the README more accessible for first timers

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactoring (cleanup of codebase withouth changing any existing functionality)

## Checklist:

- [ ] I've added tests for my code
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: features/README.md [hunk 1]
```diff
@@ -48,45 +48,28 @@ Or install it yourself as:
...
-   aruba init --test-framework cucumber
-   aruba init --test-framework minitest
+   git clone https://github.com/cli-testing/aruba-getting-started.git
+   cd aruba-getting-started
    ~~~
 
-### Your First Tests with "aruba"
-
-1. Clone the "Getting Started" application and install its dependencies
+2. Install the required depencies
```
Comment (olleolleolle) [inline features/README.md]: Typo `depencies` => `dependencies`

File: features/README.md [hunk 2]
```diff
@@ -116,10 +99,9 @@ prefer to setup `aruba` yourself, please move on to the next section.
    bundle exec cucumber
    ~~~
 
-#### RSpec
+### RSpec
 
-1. Create a file named "spec/spec_helper.rb" with the following content. If the
-   file already exists add the line to the file.
+1. Add the following line to the "spec/spec_helper.rb"-file.
```
Comment (olleolleolle) [inline features/README.md]: Remove the hyphenation in the last word. `-file` => ` file`

File: features/README.md [hunk 3]
```diff
@@ -151,9 +133,9 @@ prefer to setup `aruba` yourself, please move on to the next section.
    bundle exec rspec
    ~~~
 
-#### Minitest
+### Minitest
 
-1. Add a file named "test/test_helper.rb" with:
+1. Add the following line to the "test/test_helper.rb"-file.
```
Comment (olleolleolle) [inline features/README.md]: There is an extra `-` to remove here, as well.

File: features/README.md [hunk 4]
```diff
@@ -189,12 +169,34 @@ prefer to setup `aruba` yourself, please move on to the next section.
...
 
-## Copyright
+# Initialize an existing project
+
+There's an initializer to make it easier for you to getting started. If you
+prefer to setup `aruba` yourself, please move on to the next section.
+
+1. Go to your project's directory
+
+2. Make sure, it's under version control and all changes are committed to your
```
Comment (olleolleolle) [inline features/README.md]: `Make sure, it's ` => `Make sure it's `

**Comment by olleolleolle** [comment]
Cool change! 

I read the text and made tiny notes where I found things that can change.

**Comment by maxmeyer** [comment]
@olleolleolle Fixed. Did you use the review for your comments?

**Comment by olleolleolle** [comment]
@maxmeyer No, I thought I was only adding "loose commentary" so I made Single Comments of each.

File: features/README.md [hunk 5]
```diff
@@ -151,9 +133,9 @@ prefer to setup `aruba` yourself, please move on to the next section.
    bundle exec rspec
    ~~~
 
-#### Minitest
+### Minitest
 
-1. Add a file named "test/test_helper.rb" with:
+1. Add the following line to the "test/test_helper.rb" file.
```
Comment (mvz) [inline features/README.md]: I don't think `test/test_helper.rb` exists in the sample repo, does it?

**Comment by mvz** [comment]
I'll try out the instructions as per this PR again tonight.

File: features/README.md (hunk 5 above)
Comment (maxmeyer) [inline features/README.md]: Sorry, I forgot to push that commit. Shall be now online.

**Pull request merged**